src/otx/v2/api/entities/utils/data_utils.py

```diff
@@ -156,15 +158,38 @@ def get_old_new_img_indices(
     """
     ids_old, ids_new = [], []
     _dataset_label_schema_map = {label.name: label for label in labels}
-    new_classes = [_dataset_label_schema_map[new_class] for new_class in new_classes]
+    new_classes: list[int] = [int(_dataset_label_schema_map[new_class].id) for new_class in new_classes]
     for i, item in enumerate(dataset):
-        if item.annotation_scene.contains_any(new_classes):
+        labels = get_labels(item)
+        contain_new_class = False
+        for cls in new_classes:
```

Comment:
Sorry for bothering you, but `cls` is used for first argument of class method. Please rename it.